bin/lesson_check.py

```diff
@@ -8,8 +8,8 @@
 import re
 from argparse import ArgumentParser
 
-from util import (Reporter, read_markdown, load_yaml, check_unwanted_files,
-                  require)
+from util import *
```

Comment:
For future-me, who is not terribly well-versed in Python, this will be helpful to remind me of what `import *` is doing here.

```suggestion
# This uses the `__all__` list in `util.py` to determine what objects to import 
# see https://docs.python.org/3/tutorial/modules.html#importing-from-a-package
from util import *
```